sample_app/spark_client.py

```diff
@@ -0,0 +1,54 @@
+from pyspark.sql import SparkSession
+import os
+
+class SparkClient:
+    def __init__(self):
+        self.spark = SparkSession.builder \
+            .master("local[*]") \
+            .appName("Try RouterFS") \
+            .config('spark.sql.shuffle.partitions', '1') \
```

Comment:
I think we don't need this.  If indeed so, we should add a comment explaining that this is merely to make things run faster on this small example.  I don't want anyone to think that this is somehow required!